include/zephyr/dt-bindings/usb-c/pd.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2022 The Chromium OS Authors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * This is based on Linux, documentation:
+ *  https://github.com/torvalds/linux/blob/master/include/dt-bindings/usb/pd.h
```

Comment:
Linking to master is really bad idea. You can use https://github.com/torvalds/linux/blob/v5.19/include/dt-bindings/usb/pd.h instead (if you based it on 5.19)

The github links are shorter than git.kernel.org so you might want to use the github link in usb-c-connector.yaml (and commit message) as well.

Reply:
Done